lib/apiScenario/logger.ts

```diff
@@ -8,7 +8,7 @@ export const logger = winston.createLogger({
         winston.format.colorize({
           message: true,
         }),
-        winston.format.printf((info) => info.message)
+        winston.format.printf((info) => info.message as string)
```

Comment:
Unsure why this change was needed.  Likely a new dep version requires stronger typing.